src/which.js

```diff
@@ -2,6 +2,9 @@ var common = require('./common');
 var fs = require('fs');
 var path = require('path');
 
+// XP's system default value for PATHEXT system variable
```

Comment:
I don't see much harm in the default value. I agree that WinXP doesn't need to be supported by us, since the OS is end-of-life by Microsoft. However, just in case it's perhaps possible to delete environmental variables in Windows (is this even possible?), It'd be good to keep some fall-back value.